Extension/ui/settings.html

```diff
@@ -322,7 +323,7 @@
       <div class="main-title">IntelliSense Configurations</div>
       <div style="color: var(--vscode-foreground);">
         Use this editor to edit basic IntelliSense settings defined in the underlying <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file">c_cpp_properties.json</a> file.
-        Only the setting values of the active configuration can be modified at a time. Switch active configurations to modify a different configurations on this editor. To edit multiple configurations or set additional settings go to <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file">c_cpp_properties.json</a>.
+        Only the setting values of the active configuration can be modified at a time. Change the active configuration to modify a different configuration on this editor. To edit multiple configurations or set additional settings go to the <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file">c_cpp_properties.json</a> file.

```

Comment:
This is what I suggested in the other review:
`Changes made in this editor only apply to the active configuration. To modify other configurations <a href="...">change the active configuration</a>. To edit multiple configurations at once, or additional settings not shown here, go to <a href="...">c_cpp_properties.json</a>`

Reply:
Updated description